opwen_email_server/utils/email_parser.py

```diff
@@ -97,6 +142,41 @@ def _get_image_type(response: Response, url: str) -> Optional[str]:
     return content_type
 
 
+def _get_max_sizes() -> Tuple:
+    max_width = config.MAX_WIDTH_IMAGES
+    max_height = config.MAX_HEIGHT_IMAGES
+    return max_width, max_height
+
+
+def _is_size_already_small(size: Tuple) -> bool:
+    width = size[0]
+    height = size[1]
+    max_width = _get_max_sizes()[0]
+    max_height = _get_max_sizes()[1]
+    if width <= max_width and height <= max_height:
+        return True
+    return False
+
+
+def _change_image_size(image_content_b64: str) -> str:
+    image_content_bytes = b64decode(image_content_b64)
+    image_bytes = BytesIO(image_content_bytes)
+    image_bytes.seek(0)
+    image = Image.open(image_bytes)
+    image_format = image.format
+    is_already_small = _is_size_already_small(image.size)
```

Comment:
Nit: same as above: can just be inlined. `if _is_size_already_small(image.size)` is just as easy to understand and with less visual clutter than having this on two lines.